docs/FAQ.md

```diff
@@ -77,6 +77,11 @@ encoding in your terminal. To enable it by default, add this to the end of you `
 export LANG=en_US.utf-8
 ```
 
+## Unable to render symbols
+If some symbols don't render quite right on your terminal, it could likely be because of the symbols not being supported on the emulator/font.
+
+You can run `zulip-term-check-symbols` to check whether or not the symbols render properly on your terminal emulator.
```

Comment:
I'd suggest we combine this with the previous section?

Reply:
Yeah, that makes sense, I'll do that. I put it in a separate section thinking it would add emphasis on the tool we're providing but that makes sense.